tester_app/config/initializers/react_on_rails.rb

```diff
@@ -0,0 +1,21 @@
+# Shown below are the defaults for configuration
+ReactOnRails.configure do |config|
```

Comment:
@justin808 this makes it sound as though all of these are able to be specified via the `render_component` helper, which isn't true